modules/cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class.adoc

```diff
@@ -6,13 +6,51 @@
 [id="cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class_{context}"]
 = Creating a pod with a guaranteed QoS class
 
-Keep the following in mind when you create a pod that is given a QoS class of `Guaranteed`:
+You can create a pod with a quality of service (QoS) class of `Guaranteed` for high-performance workloads. Configuring a pod with a QoS class of `Guaranteed` ensures that the pod has priority access to the specified CPU and memory resources. 
 
-* Every container in the pod must have a memory limit and a memory request, and they must be the same.
-* Every container in the pod must have a CPU limit and a CPU request, and they must be the same.
+To create a pod with QoS class of `Guaranteed`, you must apply the following specifications:
 
-The following example shows the configuration file for a pod that has one container. The container has a memory limit and a memory request, both equal to 200 MiB. The container has a CPU limit and a CPU request, both equal to 1 CPU.
+* Set identical values for the memory limit and memory request fields for each container in the pod.
+* Set identical values for CPU limit and CPU request fields for each container in the pod.
 
+[NOTE]
+====
+A pod with a QoS class of `Guaranteed` cannot use resources beyond the specified limits.
+====
+
+The pod will not be evicted from a node unless one of the following criteria is met:
+
+* During resource contention on a node, there are no lower-priority pods that can be evicted.
```

Comment:
This is probably accurate, but rather strong, because this can only happen if the platform itself gets starved (kubelet etc.).

The docs say exactly this:

> These pods will never be evicted because of another pod's resource consumption. If a system daemon (such as kubelet and journald) is consuming more resources than were reserved via system-reserved or kube-reserved allocations, and the node only has Guaranteed or Burstable pods using less resources than requests left on it, then the kubelet must choose to evict one of these pods to preserve node stability and to limit the impact of resource starvation on other pods. In this case, it will choose to evict pods of lowest Priority first.

Reply:
Thanks for the info. I've updated it this line, but happy to delete it entirely if you think it's not worthwhile